package.json

```diff
@@ -119,7 +119,10 @@
   "scripts": {
     "build": "tsx scripts/build.ts",
     "catwalk": "tsx scripts/catwalk.ts",
-    "icons": "tsx scripts/icons.ts",
+    "icons:generate": "tsx scripts/icons/generate.ts",
+    "icons:optimize": "tsx scripts/icons/optimize.ts",
+    "icons:preview": "tsx scripts/icons/preview.ts",
+    "icons": "pnpm run '/^icons:.*/'",
```

Comment:
Do we know the order of the scripts that are ran? Is it the order in which they're defined in the scripts block,  alphabetical order, etc?

Reply:
Good point! I'll look into that ahah.

Reply:
Discussed on discord, it's the order defined in the script block